hasjob/static/js/app.js

```diff
@@ -184,6 +184,199 @@ window.Hasjob.PaySlider.prototype.resetSlider = function(currency) {
 };
 
 $(function() {
+    //Change site button to filter icon
+  $(".hg-site-nav-toggle").find("i").removeClass("fa-bars").addClass("fa-search");
+  $("#hg-sitenav").on("shown.bs.collapse", function(){
+    $(".hg-site-nav-toggle").find("i").removeClass("fa-search").addClass("fa-close");
+  });
+  $("#hg-sitenav").on("hidden.bs.collapse", function(){
+    $(".hg-site-nav-toggle").find("i").removeClass("fa-close").addClass("fa-search");
+  });
+
+  //Add Cancel button for dropdown filter in mobile
+  $("#job-filters-cancel").click(function(event){
+    event.preventDefault();
+    $("#hg-sitenav").collapse("toggle");
+  });
+  
+  var scrollheight = $("nav").height() - $(".hg-site-nav").height();
+  $(window).scroll(function(){
+    if($(window).width() > 767){
+      if ($(this).scrollTop() > scrollheight){
+        $(".header-section").slideUp();
+      }
+      else{
+        $(".header-section").slideDown();
+      }
+    }
+  });
+
   window.Hasjob.JobPost.handleGroupClick();
   $(".pstar").off().click(window.Hasjob.JobPost.handleStarClick);
+
+  if (window.location.pathname == "/"){
+    $(".search-text").hide();
+  }
+  else{
+    $(".filter-text").hide();
+  }
+
+  var query = [], parameters;
+  var filters = window.location.href.slice(window.location.href.indexOf('?') + 1).split('&');
+  for(var i = 0; i < filters.length; i++)
+  {
+      parameters = filters[i].split('=');
+      if (parameters[0] == 'currency' || parameters[0] == 'pmin' || parameters[0] == 'pmax'){
+        query[parameters[0]] = parameters[1];
+      }
+  }
```

Comment:
Please keep the styling consistent. 
```javascript
    for(var i = 0; i < filters.length; i++) {
        ...
    }
```

This is a useful guide to refer to:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Control_flow_and_error_handling